lib/src/types/index.ts

```diff
@@ -0,0 +1,50 @@
+import PolicyTypeEnum from '../nanotdf/enum/PolicyTypeEnum';
+
+export type InputSource = ReadableStream | Buffer | string | ArrayBuffer | NodeJS.ReadableStream;
+
+type Header = {
+  magicNumberVersion: string[];
+  kas: {
+    protocol: number;
+    length: number;
+    body: string;
+  };
+  eccBindingMode: {
+    useECDSABinding: boolean;
+    ephemeralCurveName: number;
+  };
+  symmetricPayloadConfig: {
+    hasSignature: boolean;
+    signatureCurveName: number;
+    symmetricCipher: number;
+  };
+  ephemeralPublicKey: string[];
+};
+
+type Policy = {
```

Comment:
<img width="716" alt="image" src="https://user-images.githubusercontent.com/41641307/183372957-a75dd2d4-b249-4c90-845f-a72f8962c816.png">
Already have this type. Check other types for duplecations, theres a lot in models defined already